quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-site/prereqs/prereq.azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "value": "eastus"
```

Comment:
remove this so we can test the defaultValue - if the RP is not globally available remove the defaultValue